src/test/java/com/kodilla/ecommercee/domain/ProductRepositoryTestSuite.java

```diff
@@ -89,11 +99,11 @@ public void testProductRepositoryUpdate() {
         //When
         groupRepository.save(group);
         String updatedName = "updated product";
-        product.setName(updatedName);
+        product = new Product(group, updatedName);
         productRepository.save(product);
 
         //Then
-        assertEquals(updatedName, product.getName());
+        assertEquals(updatedName, productRepository.findById(product.getId()).orElse(new Product()).getName());
```

Comment:
Jest prostsze rozwiązanie - jak masz chwilę wcześniej `productRepository.save(product);` to przypisz wynik metody save(), która zwraca zaktualizowaną encję z bazy danych, do zmiennej i ją przekaż do asercji